file/store.go

```diff
@@ -78,11 +81,20 @@ func (store *Store) PutFile(reader io.Reader, filename string, isPublished bool)
 			"name":   filename,
 		})
 
-		result, err := store.uploader.Upload(&s3manager.UploadInput{
+		var result *s3manager.UploadOutput
+		var err error
```

Comment:
Think about initialising `result` and `err` before they are used.

Line 97 could be the following:
`result, err := store.uploader.Upload(params)`
resulting in lines 84 and 85 redundant.

Reply:
this has been addressed in the changes on cmd-develop